src/commands/cicdPipeline.ts

```diff
@@ -0,0 +1,60 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import * as utils from 'util';
+import AksClusterTreeItem from '../tree/aksClusterTreeItem';
+import { IActionContext } from 'vscode-azureextensionui';
+
+const deployToAzureExtensionId = 'ms-vscode-deploy-azure.azure-deploy';
+const configurePipelineCommand = 'configure-cicd-pipeline';
+const browsePipelineCommand = 'browse-cicd-pipeline';
+
+export async function configurePipeline(context: IActionContext, target: any): Promise<void> {
+    if (await isDeployToAzureExtensionInstalled()) {
+        const cluster = await getCluster(target);
```

Comment:
💡 📓 idea share: seems like this code is repetitive:

* we can easily move this to a private funct, and then call it from both `configurePipeline` and `browsePipeline` .



```
if (await isDeployToAzureExtensionInstalled()) {
        const cluster = await getCluster(target);
            if (!!cluster) {
await executeDeployToAzureExtensionInstalled(browsePipelineCommand, cluster);
        }
    }
```

Reply:
Agreed.
Have made changes accordingly.